tests/index.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
```

Comment:
this is the test client. i could be swayed to rename it if it makes better sense semantically.